src/styles/index.scss

```diff
@@ -1,3 +1,88 @@
+$background-color: #eee;
+$day-size: 98px;
+$border-width: 1px;
+$day-size-with-border: $day-size + ($border-width * 2);
+$border-color: #000;
+$font-size: 30px;
+$blocks-per-row: 7;
+$calendar-width: $blocks-per-row * $day-size-with-border +
+  (($blocks-per-row - 1) * 1px);
+$margin-left-for-sun: ($blocks-per-row - 1) * $day-size-with-border +
+  ($blocks-per-row - 1) * 1px;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+@each $start-day in mon, tue, wed, thu, fri, sat, sun {
+  $margin-left: 0;
+
+  @if $start-day == sun {
+    $margin-left: $margin-left-for-sun;
+  } @else if $start-day == mon {
+    $margin-left: 0;
+  } @else if $start-day == tue {
+    $margin-left: ($day-size-with-border + 1px) * 1;
+  } @else if $start-day == wed {
+    $margin-left: ($day-size-with-border + 1px) * 2;
+  } @else if $start-day == thu {
+    $margin-left: ($day-size-with-border + 1px) * 3;
+  } @else if $start-day == fri {
+    $margin-left: ($day-size-with-border + 1px) * 4;
+  } @else if $start-day == sat {
+    $margin-left: ($day-size-with-border + 1px) * 5;
+  }
+
+  .calendar--start-day-#{$start-day} {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-width;
+    margin: 0 auto;
+    gap: 1px;
```

Comment:
This styles should be used for class without modifier like calendar and outside each.